### PR TITLE
Add video series or movie info - Iss14 [WIP]

### PR DESCRIPTION
**Pull request by xavierfav**
this PR implements what is described in #14.
The branch was created from #15 which should not get much more changes before being merged to master.

Movie or Series title, episode and season numbers are available from the API for a video this way:
```python
{
    ...
    "name": "The.Big.Bang.Theory.S05E19.HDTV.x264-LOL.mp4",
    "series": "The Big Bang Theory",
    "movie": null,
    "episode": 19,
    "season": 5
}
```

We should re-think how the interface and the search works.
We could for instance search on movies and series titles and displays the results in two different carousels.
When selecting a serie, another carousel (or maybe two) should allow to select the season and episode.
We should probably sketch some scenarios to find the most efficient/fast way to retrieve our videos.
Also in terms of implementation, what would be more simple for @worms19.

### CODE REVIEW

File: backend/StreamServerApp/models.py [hunk 1]
```diff
@@ -1,6 +1,20 @@
 from django.db import models
 
 
+class Series(models.Model):
+    title = models.CharField(max_length=200)
```
Comment (DerouineauNicolas) [inline backend/StreamServerApp/models.py]: We'll have to find a way to attribute a thumbnail to the Series as well. Maybe pick a random one ?

Comment (xavierfav) [reply backend/StreamServerApp/models.py]: Ok. Would you like this random pick to be done once when populating the db, or every time we would need a thumbnail?

**Comment by xavierfav** [comment]
Yes there would be work to do on providing ways to interact with this new database tables.
I am waiting we validate the proposed UI to start working on that.

File: backend/StreamServerApp/models.py (hunk 1 above)
Comment (DerouineauNicolas) [reply backend/StreamServerApp/models.py]: I guess it's ok if we do it once. But maybe we need one per season (if we are using a carrousel for seasons as well).

Comment (xavierfav) [reply backend/StreamServerApp/models.py]: Ok.
I think in the end we will not use a carousel per season. See the original issue for the discussion.

**Pull request merged**